libsel4/sel4_arch_include/aarch64/sel4/sel4_arch/objecttype.h

```diff
@@ -8,7 +8,6 @@
 
 typedef enum _mode_object {
     seL4_ARM_HugePageObject = seL4_NonArchObjectTypeCount,
-    seL4_ARM_PageUpperDirectoryObject,
-    seL4_ARM_PageGlobalDirectoryObject,
+    seL4_ARM_VSpaceObject,
```

Comment:
It looks like a mode-agnostic object.

Reply:
The concept could indeed be generalised to other architectures, but I would leave that for a later stage.